tests/db_integrity/test_data_updates.py

```diff
@@ -511,10 +471,9 @@ def test_obs_data_representative_date_overlap():
 
     This test checks this will no longer raise a KeyError based on this.
     """
-
-    clear_test_stores()
-    bsd_data_read_crds_internal_overlap()
-    bsd_data_read_crds_internal_overlap(overwrite=True)
+    # Add same data twice, overwriting the second time
+    bsd_crds_overwrite(False)
```

Comment:
I feel the previous setup for this was more clear - this was a test for a specific bug we found rather than a generic overwrite of crds data.

Reply:
I put the tests with overwrite parameters back as they were